_includes/footer.html

```diff
@@ -3,7 +3,7 @@
     <div class="row">
       <div class="col-md-4">
         <p class="copyright">
-          © 2021 Stargate Project Authors
+          © 2023 Stargate Project Authors
```

Comment:
or simply to never update again.. has a drawback that you need JS supported browser, but then who does not.. Needs testing :small_blue_diamond: 

`<script>document.write(new Date().getFullYear())</script>`